python/mpt/src/mpt/utils.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+from ethereum.cancun.trie import (
+    BranchNode,
+    ExtensionNode,
+    InternalNode,
+    LeafNode,
+    bytes_to_nibble_list,
+)
```

Comment:
should we already create a `cancun` module? as pectra is coming?

Reply:
Here are all the EIPs included in pectra https://eips.ethereum.org/EIPS/eip-7600

should we first look briefly if it appears pectra will modify the behaviour of the trie? if not, we might not have to add complexity just yet to the module structure

Reply:
cc @enitrat about pectra migration preparation